src/main/resources/application-han.yml

```
@@ -0,0 +1,13 @@
+  config:
+    activate:
+      on-profile: han
+
+  datasource:
+    url: jdbc:mysql://localhost:3306/medi?useSSL=false&serverTimezone=Asia/Seoul&allowPublicKeyRetrieval=true
+    username: root
+    password: springproject
+    driver-class-name: com.mysql.cj.jdbc.Driver
+  jpa:
+    hibernate:
+      ddl-auto: update
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)